server/liveblog/themes/themes.py

```diff
@@ -456,8 +456,8 @@ def _save_theme_settings(self, theme, previous_theme):
                 else:
                     # Otherwise we keep the settings that are already on the theme.
                     default_theme_settings[key] = default_prev_theme_settings[key]
-
             theme_settings.update(default_theme_settings)
+            theme_settings.update(default_prev_theme_settings)
```

Comment:
new settings and new defaults will now be updated because it is updating with the previous defaults.